examples/all_in_one/nginx/conf/nginx.conf

```diff
@@ -0,0 +1,39 @@
+events {
+  worker_connections  1024;
+}
+
+http {
+    server_tokens off;
+    include       mime.types;
+    charset       utf-8;
+
+    log_format upstream_time_with_content_class_slo_info  '$remote_addr - $remote_user [$time_local] '
+                                                'request-method="$request_method" request-uri="$request_uri" request_query="$args"'
+                                                ' $status $body_bytes_sent '
+                                                '"$http_referer" uag="$http_user_agent" '
+                                                '"$http_x_forwarded_for" ua="$upstream_addr" '
+                                                'rt="$request_time"';
```

Comment:
Isn't be easier to use default nginx logline format? From my point of view it could be faster startup for new combers.

Reply:
Default nginx format `comibned` does not include `$request_time` which we need for latency SLO computation.

However I've log format `upstream_time`, which is also included as an example in [nginx documentation](https://docs.nginx.com/nginx/admin-guide/monitoring/logging/), so I believe that many users are familiar with it.

Done within https://github.com/seznam/slo-exporter/pull/4/commits/36349550ba69f6dd981c2a9ba0af661d0feb2a1f